app/dao/service_email_reply_to_dao.py

```diff
@@ -68,7 +68,11 @@ def archive_reply_to_email_address(service_id, reply_to_id):
     reply_to_archive = ServiceEmailReplyTo.query.filter_by(id=reply_to_id, service_id=service_id).one()
 
     if reply_to_archive.is_default:
-        raise ArchiveValidationError("You cannot delete a default email reply to address")
+        non_archived_reply_tos = dao_get_reply_to_by_service_id(service_id)
+        if len(non_archived_reply_tos) > 1:
+            # this error should not be displayed in Admin, only if a user manually sends a post request
+            raise ArchiveValidationError("You cannot delete a default email reply to address if other reply to addresses exist")
```

Comment:
Hmm based on the [last comment on the issue explaining the solution](https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/684), I thought we would allow deletion of the default reply to address if others exists? I must misunderstand the code here, it seems we prevent it if other addresses exist, no?

Reply:
On Admin, I am switching the default to be one of the other reply-to addresses before deleting the old default: https://github.com/cds-snc/notification-admin/pull/1398/files#diff-a87e3297d1ea8866e40da7b2861cd7d5d8a604e7847705ebb8d5d455ff6a6877R741-R750